etc/schema.json

```diff
@@ -6542,52 +6547,68 @@
                                     "additionalProperties": false,
                                     "properties": {
                                         "frag_ignored": {
-                                            "type": "integer"
+                                            "type": "integer",
+                                            "description": "Number of IPv4 fragments ignored due to internal error"
                                         },
                                         "frag_overlap": {
-                                            "type": "integer"
+                                            "type": "integer",
+                                            "description": "Number of IPv4 fragments with overlapping data"
                                         },
                                         "frag_pkt_too_large": {
-                                            "type": "integer"
+                                            "type": "integer",
+                                            "description": "Number of IPv4 fragments ignored due to being too large"
                                         },
                                         "hlen_too_small": {
-                                            "type": "integer"
+                                            "type": "integer",
+                                            "description": "Number of IPv4 packets flagged invalid due to header smaller than minimum size"
                                         },
                                         "icmpv6": {
-                                            "type": "integer"
+                                            "type": "integer",
+                                            "description": "Number of IPv4 packets flagged invalid due to having an ICMPV6 header"
                                         },
                                         "iplen_smaller_than_hlen": {
-                                            "type": "integer"
+                                            "type": "integer",
+                                            "description": "Number of IPv4 packets flagged invalid due to length being smaller than IP header size"
                                         },
                                         "opt_duplicate": {
-                                            "type": "integer"
+                                            "type": "integer",
+                                            "description": "Number of IPv4 packets with duplicated IP options"
                                         },
                                         "opt_eol_required": {
-                                            "type": "integer"
+                                            "type": "integer",
+                                            "description": "Number of IPv4 packets with 'end of list' option required in IP options"
```

Comment:
key is that the EOL should be there, but isn't

Reply:
btw code has several EOL related todos, we should track them in a ticket I think